### PR TITLE
Update proprocessing

### PR DESCRIPTION
**Pull request by Jieran-S**
Since our transform data option is default log1p transformation. I added an additional scaling step for [GraphST](https://github.com/SpatialHackathon/SpaceHack2023/compare/update_Proprocessing?expand=1#diff-3b02763195d6bac92c0da41eb06024176fc8015d394f58eaca1cb9ee1d6df6b4) and [SCAN-IT](https://github.com/SpatialHackathon/SpaceHack2023/compare/update_Proprocessing?expand=1#diff-fadc3eb755701210f3f5e99e3e996d8165f60eeb7a28b9615d3ebc32e6a35e90). Using parameters in their source codes. 

GraphST: https://github.com/JinmiaoChenLab/GraphST/blob/main/GraphST/preprocess.py
SCAN-IT: https://github.com/zcang/SCAN-IT/blob/main/examples/Slide-seq/scanit.ipynb

For reviewers:

1. These are the only 2 methods I found using scaled data. Is there any other method I miss?

2. BANKSY requires a lib-norm (not log-norm) preprocessing. What should we do about that?

### CODE REVIEW

**Review by shdam — APPROVED** [review]

**Comment by shdam** [comment]
I think it makes sense to update Banksy to preprocess as they recommend. 
I have changed the script accordingly.

**Comment by shdam** [comment]
Thinking about it, I also found a way to use the existing preprocessing workflow in bass - the script has been updated accordingly

File: method/BANKSY/banksy.r [hunk 1]
```diff
@@ -168,8 +168,14 @@ k_geom <- config$k_geom
 npcs <- config$npcs
 resolution <- config$resolution
 method <- config$method
-assay_name <- "logcounts"
+assay_name <- "normcounts"
 set.seed(seed)
+
+# Normalization to mean library size
+spe <- computeLibraryFactors(spe)
+assay(spe, aname) <- normalizeCounts(spe, log = FALSE)
```
Comment (Jieran-S) [inline method/BANKSY/banksy.r:176]: `aname` is not defined elsewhere? Should this be `assay_name`?

Comment (shdam) [reply method/BANKSY/banksy.r]: Good catch!

**Comment by niklasmueboe** [comment]
For now we can leave it as implemented in this PR. In the future we might want to try out library size normalization as a preprocessing step in general anyway.

**Review by niklasmueboe — APPROVED** [review]

**Pull request merged**